### DISCUSSION TITLE
DietPi-Software | Pi.Alert

### DISCUSSION THREAD

**Discussion by AdamF100001** [MichaIng/DietPi#6358] [Software Request]
Please can you add Pi Alart to the software repository.

Link https://github.com/pucherot/Pi.Alert

Thanks

**Comment by holocronology** [comment]
I think this would be a more appropriate version: https://github.com/leiweibau/Pi.Alert/

**Reply by MichaIng** [reply]
Indeed, the original repo has not seen any commit in 2 years, not a great basis for newly implementing a software title. The most trusted (starts/age) fork however seems to be this one: https://github.com/jokob-sk/Pi.Alert

**Reply by holocronology** [reply]
Agreed, but isn't that one docker only?

**Reply by MichaIng** [reply]
Right, but that does not mean that it cannot be installed bare metal. There is no install script on the Docker-ready fork, but the one from the other fork is crap for our need anyway. We would anyway install that PHP app and Python backend manually.

**Reply by holocronology** [reply]
I would be happy to beta test...

**Reply by AdamF100001** [reply]
Once it's beta tested, will it be implemented into DietPi?

**Reply by Joulinar** [reply]
There is no ETA for the software title to become available. Best would be if someone could start implementing.

**Reply by NossieUK** [reply]
Just wanted to add a +1 to this, seems to not support unbound and it screws up lighttpd - been trying to find a solution to install it but each way I've tried so far has caused problems, shame as it's an invaluable tool for pihole

https://github.com/leiweibau/Pi.Alert/

The one above is not containered but there is one that is

**Reply by HyperCriSiS** [reply]
Cool project. There is a very active fork that has even HA integration now!
[https://github.com/jokob-sk/Pi.Alert](url)
